### ISSUE TITLE
PT006 meaning

### ISSUE THREAD

**Issue by sergiowalls** [m-burst/flake8-pytest-style#8]
* Python version: 3.6
* flake8-pytest-style version used: 0.6.0

### Description

Running flake8 I'm getting PT006 for a piece of code that I think it's right. Can you explain why is this happening?

### What I Did

The code:
```python3
@pytest.mark.parametrize("date_range, expected", [
    (("2019-01-20", "2019-01-20"), ["2019-01-20"]),
    (("2019-01-20", "2019-01-21"), ["2019-01-20", "2019-01-21"]),
    (("2019-01-20", "2019-01-22"), ["2019-01-20", "2019-01-21", "2019-01-22"]),
    (("2020-02-28", "2020-03-01"), ["2020-02-28", "2020-02-29", "2020-03-01"])
])
def test_get_date_range(date_range, expected):
    start, end = date_range

    assert get_date_range(start, end) == expected
```

The configuration:
```
[flake8]
max-line-length = 120
```

The execution:
```bash
pipenv run flake8
Loading .env environment variables…
artifacts/src/test_date_range.py:6:2: PT006 wrong name(s) type in @pytest.mark.parametrize, expected tuple
```

**Comment by m-burst** [comment]
Hi @sergiowalls,

PT006 tells you that you are not using the right type for the argument names in your `@pytest.mark.parametrize`. The general intention of the rule is to enforce consistency between all tests in a single codebase, i.e. to disallow using different types in different tests.

With default settings, your code should be:
```py
@pytest.mark.parametrize(("date_range", "expected"), [  # <- note the tuple
    ...
])
def test_...():
    ...
```

If you prefer to specify argument names as a single comma-separated string, you can set `pytest-parametrize-names-type` configuration option to `csv`. See the [readme](https://github.com/m-burst/flake8-pytest-style#configuration) for more details.
